src/components/patterns/blog-details/BlogDetails.vue

```diff
@@ -0,0 +1,73 @@
+<template>
+    <div
+        class="vs-blog-details"
+        data-test="vs-blog-details"
+    >
+        <span class="vs-blog-details__author">
+            {{ blogAuthor }} &nbsp;&nbsp;·&nbsp;&nbsp;
```

Comment:
Can this be achieved with margins/padding?

Reply:
Yes, I'll update, this was leftover from me doing it in the browser with Maria :D